integration-test-groups/azure/azure-eventhubs/src/test/java/org/apache/camel/quarkus/component/azure/eventhubs/it/AzureEventhubsTest.java

```diff
@@ -16,45 +16,422 @@
  */
 package org.apache.camel.quarkus.component.azure.eventhubs.it;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
-import org.apache.commons.lang3.RandomStringUtils;
+import io.restassured.response.Response;
 import org.awaitility.Awaitility;
-import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 @EnabledIfEnvironmentVariable(named = "AZURE_STORAGE_ACCOUNT_NAME", matches = ".+")
 @EnabledIfEnvironmentVariable(named = "AZURE_STORAGE_ACCOUNT_KEY", matches = ".+")
 @EnabledIfEnvironmentVariable(named = "AZURE_EVENT_HUBS_BLOB_CONTAINER_NAME", matches = ".+")
 @EnabledIfEnvironmentVariable(named = "AZURE_EVENT_HUBS_CONNECTION_STRING", matches = ".+")
 @QuarkusTest
 class AzureEventhubsTest {
+    // NOTE: Consumer endpoints are started / stopped manually to prevent them from inferring with each other
 
-    private static final Logger LOG = Logger.getLogger(AzureEventhubsTest.class);
+    @AfterEach
+    public void afterEach() {
+        RestAssured.post("/azure-eventhubs/reset/mocks")
```

Comment:
Same way, don't we have reasonable way of avoiding this ? Like an option to restart quarkus context on every test ?